doc/rados/operations/erasure-code.rst

```diff
@@ -27,10 +27,11 @@ requires at least three hosts::
 Erasure code profiles
 ---------------------
 
-The default erasure code profile sustains the loss of a two OSDs. It
-is equivalent to a replicated pool of size three but requires 2TB
-instead of 3TB to store 1TB of data. The default profile can be
-displayed with::
+The default erasure code profile sustains the loss of two hosts.  It
```

Comment:
i'd suggest avoid using host for OSD. they are different concepts even a decent crush map will pick different hosts or even racks for different shards for an erasure coded pool depending on how the crush map is designed.